UWB_node:
  ros__parameters:
    # ------------------ frame id ------------------
    UWB_T0_frame: "UWB_T0"    # 标签 0
    UWB_A0_frame: "UWB_A0"    # 基站 0
    UWB_A1_frame: "UWB_A1"    # 基站 1
    UWB_A2_frame: "UWB_A2"    # 基站 2
    UWB_A3_frame: "UWB_A3"    # 基站 3

    # ---------------- 标签及基站坐标 ----------------
    UWB_T0_x: 0.0     # 标签 0
    UWB_T0_y: 0.0     
    UWB_T0_z: 0.0     

    UWB_A0_x: 0.0     # 基站 0
    UWB_A0_y: 0.0
    UWB_A0_z: 0.0

    UWB_A1_x: 0.0     # 基站 1
    UWB_A1_y: 0.0
    UWB_A1_z: 0.0

    UWB_A2_x: 0.0     # 基站 2
    UWB_A2_y: 0.0
    UWB_A2_z: 0.0

    UWB_A3_x: 0.0     # 基站 3
    UWB_A3_y: 0.0
    UWB_A3_z: 0.0 
    
    # ----------------- usb 端口 -------------------
    usb_port: "/dev/ttyUSB0"  # USB端口

    # ------------------- 滤波 ---------------------
    is_fliter: True           # 是否开启滤波

    # -------------- 参数 均需为浮点数 ---------------
    update_rate:    10.0      # 最大为 50HZ
    min_distance:   0.0       # 最小距离
    max_distance:   149.0     # 最大距离
